static/lang/en.json

```diff
@@ -2990,6 +2991,11 @@
         "SpellCategorySpell": "Spell",
         "SpellUsesLabel": "Uses",
         "SpellUnlimitedLabel": "Unlimited",
+        "SpellVariantDeleteDialogTitle": "Delete Spell Variant",
+        "SpellVariantDeleteDialogText": "Are you sure you want to delete '{variantName}'?",
+        "SpellVariantSelectVariantLabel": "Select Variant",
+        "SpellVariantsLabel": "Spell Variants",
+        "SpellVariantSheetTitle": "{originalName} (Variant)",
```

Comment:
We're avoiding bloating the top-level namespace now: move all this to `PF2E.Item.Spell.Variant`